### PR TITLE
refactor to create classes and add tests

### PR DESCRIPTION
**Pull request by ebarnsli**
so i kind of borrowed code from https://github.com/ankitdobhal/MailBomber/pull/3 

my suggestion would be to adopt more structure or architecture to your script so that you can more readily change up what it does. To me the idea is to build out a class or suite of features that will help with the mail bombing. If you go for more of a class structure or more at least more organization you'll be more flexible in the long run.

I added files into a `src` folder. this is a best practice with python projects. We still have the main file that can kick off the bomber, it just keeps things organized. 

I added tests as well. I think tests are also kind of important in this scenario because it will allow you to iterate so much faster. Sending yourself a bunch of emails and checking the result gets time consuming, this will catch small errors and save time.

### CODE REVIEW

File: MailBomber.py [hunk 1]
```diff
@@ -1,53 +1,54 @@
...
-     '          '         '      '      '     '    '       '    '          '    '          '          '         '    '        '    )/       '     '   
-                                                                                                                                   '                                                                                           
-                                                     [++] MailBomber is a mass mailer script or tool [++]
-                                                                Coded By: Ankit Dobhal                                
-                                                             Let's Begin To Bomb the mails..!            
--------------------------------------------------------------------------------------------------------------------------------------------------------
-MailBomber version 1.0
-""" \
-    + END
-print(ascii_art)
```
Comment (ebarnsli) [inline MailBomber.py:28]: move to util file :)

File: src/mail_client.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import smtplib
+import time
+import getpass
+from src.mail_bomb import MailBomb
+
+
+class MailClient:
```
Comment (ebarnsli) [inline src/mail_client.py:7]: to me it makes sense to make a client wrapper that does smtpobj operations that would reoccur, so  i made a class.

**Comment by powerexploit** [comment]
I also want to you to come with this tool in more powerful way & my other tool!
Hope we will contribute in more powerful way!
Thanks

**Comment by ebarnsli** [comment]
it's kind of a trolly tool XD i hoepfully won't be email bombing people myself but i MIGHT make some actual feature contributions

File: src/mail_client.py (hunk 2 above)
Comment (powerexploit) [reply src/mail_client.py:7]: Well I am happy & impressed that you tried to add something special with class structure !

**Comment by powerexploit** [comment]
> it's kind of a trolly tool XD i hoepfully won't be email bombing people myself but i MIGHT make some actual feature contributions

Ahh!
But I am one who love to do pen testing so love these kind of stuffs,& I am happy actually want to do more contributions.
Thanks

**Comment by ebarnsli** [comment]
> > it's kind of a trolly tool XD i hoepfully won't be email bombing people myself but i MIGHT make some actual feature contributions

> But I am one who love to do pen testing so love these kind of stuffs,& I am happy actually want to do more contributions.

oh lol that makes more sense <3

**Comment by powerexploit** [comment]
> > > it's kind of a trolly tool XD i hoepfully won't be email bombing people myself but i MIGHT make some actual feature contributions
> 
> > But I am one who love to do pen testing so love these kind of stuffs,& I am happy actually want to do more contributions.
> 
> oh lol that makes more sense <3

Yes sure brother #!/bin/bash

**Pull request merged**